src/WebSerial.cpp

```diff
@@ -42,7 +56,7 @@ void WebSerialClass::begin(AsyncWebServer *server, const char* url) {
           size_t message_size = (data[4] << 8) | data[3];
           // Issue callback
           if(_recv != nullptr){
-            _recv(data + 4, message_size);
+            _recv(data + 5, message_size);
```

Comment:
`data + 4` points to the MSB of the message size...

Reply:
Yikes. I'm glad you noticed.